edx_exams/apps/api/v1/tests/test_views.py

```diff
@@ -346,3 +346,46 @@ def test_config_create(self):
 
         config = CourseExamConfiguration.objects.get(course_id=self.course_id)
         self.assertEqual(config.provider, self.test_provider)
+
+
+class ProctoringProvidersViewTest(ExamsAPITestCase):
+    """
+    Tests ProctoringProvidersView
+    """
+
+    def get_response(self):
+        """
+        Helper function to make a get request
+        """
+        url = reverse("api:v1:proctoring-providers-list")
+        response = self.client.get(url)
+        return response
+
+    def test_proctoring_providers_list(self):
+        test_provider2 = ProctoringProvider.objects.create(
+            name='test_provider2',
+            verbose_name='testing_provider2',
+            lti_configuration_id='223456789'
+        )
+        response = self.get_response()
+        response_data_list = []
+
+        for item in response.data:
+            response_data_list.append(item['name'])
+
+        response_data_list.sort()
+
+        expected_data = sorted([self.test_provider.name, test_provider2.name])
```

Comment:
this variable just gets in the way now, you could drop the two names right into the assert below